ethereum/contracts/Messages.sol

```diff
@@ -88,9 +89,10 @@ contract Messages is Getters {
             /// introduced accidentally later due to the nontrivial storage
             /// semantics of solidity.
             require(sig.guardianIndex < guardianCount, "guardian index out of bounds");
+            require(signatory != address(0));
```

Comment:
any reason why this check is not on line 80 (right after the variable declaration)?

Reply:
We can move it up and that's fair enough!